### PR TITLE
Work without ActiveStorage

### PR DESCRIPTION
**Pull request by jpawlyn**
If `ActiveStorage` is not included in a Rails app, then we see the error `uninitialized constant Avo::Fields::Common::Files::ControlsComponent::ActiveStorage`

# Description
When running  with `eager_load` set to true, when starting puma we are seeing the error:

```
3.3.3/lib/ruby/gems/3.3.0/gems/avo-3.12.0/app/components/avo/fields/common/files/controls_component.rb:10:in `<class:ControlsComponent>': uninitialized constant Avo::Fields::Common::Files::ControlsComponent::ActiveStorage (NameError)

  prop :file, ActiveStorage::Attachment
              ^^^^^^^^^^^^^
```

Fixes # (issue)

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the [documentation](https://github.com/avo-hq/avodocs)
- [ ] I have added tests that prove my fix is effective or that my feature works

## Manual review steps

1. Step 1
1. Step 2

Manual reviewer: please leave a comment with output from the test if that's the case.

### CODE REVIEW

**Comment by jpawlyn** [comment]
Note, we only see this issue with `3.11.8` and above - see https://github.com/avo-hq/avo/compare/v3.11.7...v3.11.8

File: app/components/avo/fields/common/files/controls_component.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Avo::Fields::Common::Files::ControlsComponent < Avo::BaseComponent
   delegate :id, to: :@field
 
   prop :field, Avo::Fields::BaseField
-  prop :file, ActiveStorage::Attachment
+  prop :file, ActiveStorage::Attachment if defined?(ActiveStorage::Attachment)
```
Comment (Paul-Bob) [inline app/components/avo/fields/common/files/controls_component.rb:10]:
```suggestion
  prop :file, _Any
```

File: app/components/avo/fields/common/files/view_type/grid_item_component.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 class Avo::Fields::Common::Files::ViewType::GridItemComponent < Avo::BaseComponent
   prop :field, Avo::Fields::BaseField
   prop :resource, Avo::BaseResource
-  prop :file, _Nilable(ActiveStorage::Attachment)
+  prop :file, _Nilable(ActiveStorage::Attachment) if defined?(ActiveStorage::Attachment)
```
Comment (Paul-Bob) [inline app/components/avo/fields/common/files/view_type/grid_item_component.rb:6]:
```suggestion
  prop :file, _Nilable(_Any)
```

**Review by Paul-Bob — APPROVED** [review]
Hello @jpawlyn, thanks for this contribution!

In the original approach, if `ActiveStorage::Attachment` is not defined, the `prop :file` would also remain undefined.

I applied a suggestion to remove the type from `file` to ensure the prop exists even if `ActiveStorage::Attachment` is not defined.

**Pull request merged**